### DISCUSSION TITLE
Unable to install Tailwind CSS through VS Code terminal

### DISCUSSION THREAD

**Discussion by janoschherrmann** [tailwindlabs/tailwindcss#2059] [Help]
Hey guys,

an absolute coding beginner here, so please don't judge my stupid question:

I was trying to install Tailwind as shown by Adam in this video: [https://tailwindcss.com/course/setting-up-tailwind-and-postcss/]

I have installed Node.js in order to run the npm commands in VS Code, but when I try to run "npm init -y" I get this error message:

npm: The term "npm" was not recognized as the name of a cmdlet, function, script file or executable program. Check the spelling of the name, or whether the path is correct (if included), and
repeat the process.
In line: 1 character: 1
+ npm install tailwindcss
+ ~~~
    + CategoryInfo: ObjectNotFound: (npm: String) [], CommandNotFoundException
    + FullyQualifiedErrorId: CommandNotFoundException

Can anybody tell me what I'm doing wrong?

Thanks in advance :)

**Comment by estevanmaito** [comment]
What is returned if you run this on terminal?

```sh
node -v
```

**Reply by janoschherrmann** [reply]
I get "v12.18.2"

**Reply by janoschherrmann** [reply]
I've tested it on my laptop now as well (where I've also installed Node.js). On here, I get a different error message when typing out "npm init -y:

0 info it worked if it ends with ok
1 verbose cli [
1 verbose cli   'C:\\Program Files\\nodejs\\node.exe',
1 verbose cli   'C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js',
1 verbose cli   'init',
1 verbose cli   '-y'
1 verbose cli ]
2 info using npm@6.14.5
3 info using node@v12.18.2
4 silly package data undefined
5 info init written successfully
6 verbose stack Error: Invalid name: "Tailwind CSS Projekt"
6 verbose stack     at ensureValidName (C:\Program Files\nodejs\node_modules\npm\node_modules\normalize-package-data\lib\fixer.js:336:15)
6 verbose stack     at Object.fixNameField (C:\Program Files\nodejs\node_modules\npm\node_modules\normalize-package-data\lib\fixer.js:215:5)
6 verbose stack     at C:\Program Files\nodejs\node_modules\npm\node_modules\normalize-package-data\lib\normalize.js:32:38
6 verbose stack     at Array.forEach (<anonymous>)
6 verbose stack     at normalize (C:\Program Files\nodejs\node_modules\npm\node_modules\normalize-package-data\lib\normalize.js:31:15)
6 verbose stack     at final (C:\Program Files\nodejs\node_modules\npm\node_modules\read-package-json\read-json.js:429:5)
6 verbose stack     at then (C:\Program Files\nodejs\node_modules\npm\node_modules\read-package-json\read-json.js:161:5)
6 verbose stack     at ReadFileContext.<anonymous> (C:\Program Files\nodejs\node_modules\npm\node_modules\read-package-json\read-json.js:350:20)
6 verbose stack     at ReadFileContext.callback (C:\Program Files\nodejs\node_modules\npm\node_modules\graceful-fs\graceful-fs.js:123:16)
6 verbose stack     at FSReqCallback.readFileAfterOpen [as oncomplete] (fs.js:257:13)
7 verbose cwd C:\Users\janos\Downloads\Tailwind CSS Projekt
8 verbose Windows_NT 10.0.18363
9 verbose argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "init" "-y"
10 verbose node v12.18.2
11 verbose npm  v6.14.5
12 error Invalid name: "Tailwind CSS Projekt"
13 verbose exit [ 1, true ]

**Reply by estevanmaito** [reply]
Don't use spaces or special characters in your folders.

instead of "Tailwind CSS Projekt" use `tailwind-css-projekt`

**Reply by janoschherrmann** [reply]
Now it works :)

Thank you so much!
